src/QtAVPlayer/qavplayer.cpp

```diff
@@ -1313,6 +1313,19 @@ void QAVPlayer::setInputOptions(const QMap<QString, QString>  &opts)
     Q_EMIT inputOptionsChanged(opts);
 }
 
+
+/*!
+ * \brief Use to set log level of FFmpeg backend
+ * \param[in] level
+ * Level log to use. Please see:
+ * https://ffmpeg.org/doxygen/trunk/group__lavu__log__constants.html
+ * for value details
+ */
+void QAVPlayer::setLogsLevelBackend(int level)
+{
+    av_log_set_level(level);
```

Comment:
How you like an idea to remove this func `setLogsLevelBackend` and use `av_log_set_level` directly? 
Mean why we need to duplicate the func.

Reply:
Because all FFmpeg machinery is hide behind QtAVPlayer library, so having this method follow the same purpose (besides the fact that this method will be easier to find)